notebooks/I3C-Protocol-API/I3C-Protocol-Controller-API/I3C-Protocol-Controller-API-Basic/I3C-Protocol-Supernova-API.ipynb

```diff
@@ -830,7 +935,7 @@
     }
    ],
    "source": [
-    "TARGET_ADDR = 0x08\n",
+    "TARGET_ADDR = 0x0A\n",
```

Comment:
Use a variable to indicate the address, in this case corresponds to the `LSMD6V_DYNAMIC_ADDRESS`